src/main/java/it/pagopa/interop/probing/telemetry/writer/annotations/validator/TelemetryValidator.java

```diff
@@ -0,0 +1,21 @@
+package it.pagopa.interop.probing.telemetry.writer.annotations.validator;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import it.pagopa.interop.probing.telemetry.writer.annotations.ValidateTelemetry;
+import it.pagopa.interop.probing.telemetry.writer.dto.TelemetryDto;
+import it.pagopa.interop.probing.telemetry.writer.util.EserviceStatus;
+
+public class TelemetryValidator implements ConstraintValidator<ValidateTelemetry, TelemetryDto> {
+
+  @Override
+  public boolean isValid(TelemetryDto dto, ConstraintValidatorContext context) {
+    return Objects.isNull(dto.getStatus())
```

Comment:
How about to use Predicate for gaining in readability?

https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html